ctest: fix return value of check()

ldorau opened the pull request:
Remove 'tail' from the pipe, because it overwrites
the return value of 'match'.

vitalyvch commented on test/helper_functions.sh:
```diff
@@ -106,6 +106,13 @@ function get_files() {
 # check -- check test results (using .match files)
 #
 function check() {
+	local TEMP=$(mktemp)
```
Please replace the name of the variable with something more safer, such as MY_CHECK_TEMP.

ldorau replied:
Done

ldorau committed:
ctest: fix return value of check()

Remove 'tail' from the pipe, because it overwrites
the return value of 'match'.
